casper/src/main/scala/coop/rchain/casper/engine/Running.scala

```diff
@@ -178,59 +162,51 @@ object Running {
     casperContains(b.blockHash)
       .ifM(
         Log[F].info(s"Received block ${PrettyPrinter.buildString(b.blockHash)} again."),
-        for {
-          _      <- Log[F].info(s"Received ${PrettyPrinter.buildString(b)}.")
-          status <- casperAdd(b)
-          _ <- if (BlockStatus.isInDag(status.merge)) RequestedBlocks[F].modify(_ - b.blockHash)
-              else noop[F]
-        } yield ()
+        Log[F].info(s"Received ${PrettyPrinter.buildString(b)}.") *> casperAdd(b) >>= (
```

Comment:
Didn't we decide to stick with >> ?

Reply:
Dunno - but sure, will change, thanks

Reply:
yes, we did. `*>` is banned.

Reply:
Well, I can imagine why it's frowned upon (no evaluation order) but downright _banned_ seems harsh. I do not care actually what is the evaluation order of the log statement wrt to "action"

Reply:
Replaced in aa9d7aa394d9ee9e6a01bd3e62b89fb417e5cc86

Reply:
I was also about to ask this in some other line, but I noticed this "resolved conversation". Better do a fixup to avoid confusion